src/functionalTest/resources/features/F-017/F-017.feature

```diff
@@ -0,0 +1,85 @@
+@F-017
+Feature: F-017 : Create Role Assignments
+
+  Background:
+    Given an appropriate test context as detailed in the test data source
+
+  @S-250
+  Scenario: Invoking POST role-assignments api with request to add a new role assignment
+    Given an appropriate test context as detailed in the test data source,
+    And a user [Befta1 - who invokes the API],
+    When a request is prepared with appropriate values,
+    And the request [has roleType Organisation, roleCategory Professional, roleName Role1 and attributes has caseAccessGroupId and caseType],
+    And it is submitted to call the [Create Role Assignments] operation of [Role Assignments Service],
+    Then a positive response is received,
+    And the response has all other details as expected
+
```

Comment:
We need to delete the role assignments created for testing. Please add below step at end of sceanrio.
And a successful call [to delete role assignments just created above] as in [DeleteDataForRoleAssignments].

Reply:
This is now fixed. Please check the latest commit